pkg/miniogw/gateway-storj.go

```diff
@@ -38,7 +207,7 @@ func (s *Storj) Name() string {
 
 // NewGatewayLayer implements cmd.Gateway
 func (s *Storj) NewGatewayLayer(creds auth.Credentials) (
-	cmd.ObjectLayer, error) {
+	minio.ObjectLayer, error) {
```

Comment:
see the comment below, but we can do `return &storjObjects{s:s}` to pass in the non-global state

Reply:
Modified as suggested to eliminate global references